### DISCUSSION TITLE
Intégration de l'entreprise DeMetal dans un prochain DLC

### DISCUSSION THREAD

**Discussion by Mcfloy** [SOZ-Faut-etre-Sub/SOZ-FiveM-Server#208] [Ideas]
Actuellement DeMetal est une entreprise lié à Pole Emploi mais dans l'idée on pourrait la séparer et en faire une entreprise avec des mécaniques RP.

J'aimerais juste partager quelques idées sur ça, et après laisser le choix là dessus: tout d'abord j'imaginais DeMetal comme une entreprise lié à la Haute Criminalité, dans le sens où elle serait une entreprise qui fabriquerait les armes, que ce soit pour les FdO ou les criminels, peu importe la manière dont ils récupèrent ces armes. En terme de design on se calque sur BAUN, FFS, CM avec du crafting/farm, la subtilité ici serait de diminuer l'aspect farm et d'augmenter les interactions.

# Récupération des matériaux

Pour crafter il faut des matériaux de base, là dans l'idée DeMetal utiliserait des minerais de métaux différents selon leur provenance:
- Déchetterie Sud, zone actuelle de DeMetal, pour le fer
- Cimetière d'avions à Blaine County, pour l'acier
- Mine ouverte à l'Est du cimetière d'avions, pour l'aluminium

# Fonderie

La fonderie permet de transformer les minerais de métaux en lingots, chaque métal a son point de fusion.

La fonderie contient X fourneaux, lorsqu'on utilise un fourneau pour faire fondre le métal, une température de fusion est automatiquement mise en place, l'augmentation de la température consomme beaucoup d'électricité et prend un peu de temps. La stabilisation de la température consomme moins d'électricité, et la diminution de la température du four ne consomme pas d'électricité.

L'idée est donc que déjà il faut anticiper l'allumage du fourneau (genre 5 à 10 minutes, le temps de faire un aller retour entre la fonderie et le cimetière d'épave) mais aussi bien s'organiser pour faire les fontes de métaux en même temps pour économiser en énergie. C'est cette mécanique qui, bien qu'elle semble complexe, permettrait de vraiment donner un aspect plus original au crafting et au farming. Il faut que ce soit assez chiant pour éviter le farming intensif, mais assez ludique pour que la personne continue d'être intéressée par ce mécanisme.

Chaque métal a son point de fusion, à titre d'exemple:
- Fer: 1538°C
- Acier: 1250°C
- Aluminium: 660 °C

# Liste des recettes

- Fabrication de plaque de fonte à partir de fer
- Fabrication d'armes à feu (pistolet, fusil, mitraillettes, etc.) à partir d'acier
- Fabrication de munitions à partir d'acier ou de fer
- Fabrication de seringue à partir d'aluminium
- Fabrication de bijoux à partir d'aluminium ou d'acier ?

# Liens avec les autres entreprises

- FDO: Acquisition d'armes à feu et de munitions
- Criminels: Acquisition d'armes à feu et de munitions (soit par marchandage, soit par vol, soit autrement)
- NG: Utilisation de plaque de fonte pour les réparations de véhicule, avec une plaque qui recouvre un certain pourcentage du véhicule.
- STONK: Idéalement c'est eux qui gèrent le transfert d'armes vers les FDO, et donc qui seraient susceptibles d'être attaqués par des criminels, ce serait une source RP d'armes pour la haute criminalité, si tant est qu'ils arrivent à mettre la main dessus.

**Comment by Zocaxu** [comment]
J'aime beaucoup l'idée ! Dans l'idée, il me semble que DeMetal était déjà destiné à devenir une "vraie" entreprise. Hâte de voir ce que cela donnera. Si projet inclus du développement open source, je peux proposer mon aide :)

**Comment by NariieL** [comment]
Merci Mr Mcfloy.
